lib/spree/retailops/rop_order_helper.rb

```diff
@@ -75,8 +75,9 @@ def apply_shipment_price(price, order_level = nil)
             changed = true
           end
 
+          # Do not delete shipping tax adjustments
           if shipment.respond_to?(:adjustment_total) && shipment.adjustment_total > 0
-            shipment.adjustments.delete_all
+            shipment.adjustments.where("source_type != 'Spree::AvalaraTransaction'").delete_all
```

Comment:
We do not have pre-defined scope for `source_type != 'Spree::AvalaraTransacton''`

I can pre-define scope called `adjustment_without_tax` or something if it's preferred